frontend/app/components/Thermometer.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import styles from "./styles/Thermometer.module.scss";
+
+type Props = {
+  tasks: string;
+};
+
+const Thermometer: React.FC<Props> = ({ tasks }) => {
+  const taskStatusInPercent = { complete: 10, asPlanned: 30, unknown: 25, delayed: 30, failed: 5 };
+  const taskStatusInOrder = [
+    taskStatusInPercent.failed,
+    taskStatusInPercent.delayed,
+    taskStatusInPercent.unknown,
+    taskStatusInPercent.asPlanned,
+    taskStatusInPercent.complete,
+  ];
+  const taskStatusCumulative = taskStatusInOrder.map(
+    (
+      (sum) => (value) =>
+        (sum += value)
+    )(0),
+  );
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.thermometer}>
+        <div
+          style={{ bottom: taskStatusCumulative[4] + "%", height: taskStatusInOrder[4] + "%" }}
+          className={styles.complete}
+        ></div>
+        <div
+          style={{ bottom: taskStatusCumulative[3] + "%", height: taskStatusInOrder[3] + "%" }}
+          className={styles.asPlanned}
+        ></div>
+        <div
+          style={{ bottom: taskStatusCumulative[2] + "%", height: taskStatusInOrder[2] + "%" }}
+          className={styles.unknown}
+        ></div>
+        <div
+          style={{ bottom: taskStatusCumulative[1] + "%", height: taskStatusInOrder[1] + "%" }}
+          className={styles.delayed}
+        ></div>
+        <div
+          style={{ bottom: taskStatusCumulative[0] + "%", height: taskStatusInOrder[0] + "%" }}
+          className={styles.failed}
+        ></div>
+      </div>
+      <div className={styles.triangle}></div>
```

Comment:
Ich habe mal etwas mit dem css versucht, aber keine so gute Lösung gefunden. Wäre ein svg vielleicht eine Alternative?
`    <svg className={styles.triangleicon + ' ' + lastStyle} width={50} height={10+50}>
        <path d={'M0 0 L${50 / 2} ${10}  L${50} 0'} strokeWidth="1.5" />
      </svg>
`
      
 und styling 
`
  .failed {
    background-color: $colorFailed;
    &.triangleicon {
     background-color: transparent;
    }
    path { /*target the image with css*/
      fill: $colorFailed;
    }
  }
  
  .triangleicon {
    path {
      stroke: $thermometerBorderColor;
    }
  }
`
sah bei mir ok aus. Höhe und Breite müssen dann leider auch in die componente. LastStyle sollte davon abhängig sein was unten im Thermometer ist, als styles.failed oder andere falls eben keine gefailten tasks.